src/card/card.js

```diff
@@ -23,15 +24,15 @@ export default (data, cardStyling, sel, selectionState) => {
   const topColor = colorUtils.getDarkColor(backgroundColor);
   const fontColor = getFontColor(cardStyling, backgroundColor);
```

Comment:
This is not encoded now

Reply:
the font color is only set in the code, without user input

Reply:
That is only for default font color. What about this:

![image](https://user-images.githubusercontent.com/25658598/115406055-1a928e00-a1ef-11eb-9df2-7232c26c91e2.png)

Reply:
you're right, I didn't read the code correctly there